spin-up-hub/index.html

```diff
@@ -40,6 +40,8 @@
       rel="stylesheet"
     />
 
+    <title>Spin up Hub</title>
```

Comment:
Oh wait was that the compiled version

```suggestion
    <title>Spin Up Hub</title>
```

Reply:
fixed!